blocks/rich-text/index.js

```diff
@@ -876,7 +876,7 @@ RichText.defaultProps = {
 	format: 'element',
 };
 
-export default compose( [
+const RichTextContainer = compose( [
```

Comment:
Can we just reassign `RichText` here? `RichText = compose( [ ] )( RichText );`

Reply:
This was not possible because we use the inner RichText in unit tests.